src/styles/styles.js

```diff
@@ -2003,8 +2010,8 @@ const styles = {
 
     displayName: {
         fontSize: variables.fontSizeLarge,
-        fontFamily: fontFamily.EXP_NEUE_BOLD,
-        fontWeight: fontWeightBold,
+        fontFamily: fontFamily.EXP_NEW_KANSAS_MEDIUM,
```

Comment:
Rather than needing to explicitly call in `fontFamily.EXP_NEW_KANSAS_MEDIUM` - is there a way we can just add something similar to a Sass mixin that would basically work like a variable and pull in the fontFamily and fontWeight for us? 

Something in the Sass world would be like `@extend .headlineFont`

Basically this way we only need to maintain our headline style (font family and weight) in one place.

Reply:
Are you thinking of something like:

```
const headlineFont = {
   fontFamily: fontFamily.EXP_NEW_KANSAS,
   fontWeight: '500'
}
```

and then call `...headlineFont` in each usage, e.g.:

```
    displayName: {
        ...headlineFont,
        fontSize: variables.fontSizeLarge,
        color: themeColors.heading,
    },
```

Would that solve your concern?

Reply:
I pushed the changes above. Let me know if that's not what you had in mind and we can update it again!